vendor/src/github.com/docker/libnetwork/network.go

```diff
@@ -896,14 +905,14 @@ func (n *network) getSvcRecords(ep *endpoint) []etchosts.Record {
 	var recs []etchosts.Record
 	sr, _ := n.ctrlr.svcDb[n.id]
 
-	for h, ip := range sr {
+	for h, ip := range sr.svcMap {
 		if ep != nil && strings.Split(h, ".")[0] == ep.Name() {
 			continue
 		}
 
 		recs = append(recs, etchosts.Record{
 			Hosts: h,
-			IP:    ip.String(),
+			IP:    ip[0].String(),
```

Comment:
Off topic, but we should **really** make sure that "no longer updating `/etc/hosts`" is explicitly mentioned in the changelog/release notes. I already had someone reporting that they currently have a filewatcher on `/etc/hosts` to automate things 😕